### PR TITLE
Add release notes for SwiftPM 5.3

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

**Comment by neonichu** [comment]
@swift-ci please smoke test

File: Documentation/ReleaseNotes/5.3.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "SamplePackage",
+    dependencies: [
+        .package(url: "https://github.com/pureswift/bluetooth", .branch("master")),
```
Comment (abertelrud) [inline Documentation/ReleaseNotes/5.3.md:17]: We should probably change this to main.

**Comment by neonichu** [comment]
@swift-ci please smoke test

File: Documentation/ReleaseNotes/5.3.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# SwiftPM 5.3 Release Notes
+
+SwiftPM brings several new features in version 5.3, all of which are opt-in based on the tools-version.
+
+### Conditional dependencies
+
+You can now declare conditions for a Swift package’s target dependencies, such as limiting the dependencies by platform. This gives you more flexibility to describe complex target dependencies that support multiple platforms. (40237402)
+
+```
```
Comment (ktoso) [inline Documentation/ReleaseNotes/5.3.md:9]: Could be worth marking as swift for code highlighting of the snippet? 

```suggestion
```swift
```

**Review by ktoso — COMMENTED** [review]
A bit weird with the radar numbers here as they're not really viewable by open source community, but otherwise LGTM :)

**Comment by tomerd** [comment]
cool, maybe also add a "Release notes" section to the readme with link to this new directory?

**Comment by neonichu** [comment]
@swift-ci please smoke test

File: README.md [hunk 3]
```diff
@@ -39,6 +39,8 @@ For additional documentation on developing the Swift Package Manager itself, see
 
 For detailed documentation on the package manifest API, see [PackageDescription API](https://docs.swift.org/package-manager/PackageDescription/index.html).
 
+For release notes with information about changes between versions, see [Release Notes](ReleaseNotes).
```
Comment (tomerd) [inline README.md:42]: I think this link should be [Release Notes](Documentation/ReleaseNotes)

Comment (neonichu) [reply README.md]: Ah yes, of course.

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by neonichu** [comment]
@swift-ci please smoke test linux

**Review by tomerd — APPROVED** [review]
lgtm

**Pull request merged**